src/lib/containers/StyleGuidistComponentWrapper.tsx

```diff
@@ -15,25 +15,27 @@ import {
 export async function sessionChangeHandler() {
   detectSSOCode()
   const accessToken = await getAccessTokenFromCookie()
-  global.accessToken = accessToken
   const profile = await getUserProfile(accessToken)
-  global.currentUserProfile = profile
+  let date
   if (accessToken) {
     getAuthenticatedOn(accessToken).then(authenticatedOn => {
-      const date = moment(authenticatedOn.authenticatedOn).format('L LT')
+      date = moment(authenticatedOn.authenticatedOn).format('L LT')
     })
   }
-  return profile
+  return { accessToken, profile, authenticatedOn: date }
 }
 
 export const StyleGuidistComponentWrapper: React.FC = props => {
+  const [accessToken, setAccessToken] = React.useState(undefined)
   useEffect(() => {
-    sessionChangeHandler()
-  }, [])
+    sessionChangeHandler().then(data => {
+      setAccessToken(data.accessToken)
+    })
+  })
```

Comment:
Since we're pulling a cookie instead of inheriting a prop, we can't keep track of when the access token changes. So, let's run this check after every render.

Additionally, store it in state instead of in a global variable, so React can re-render context if it changes.